kitsune/questions/jinja2/questions/question_details.html

```diff
@@ -30,7 +30,9 @@
 
 {% block breadcrumbs %}
 
-{# {{ progress_bar(4) }} #}
+{% if aaq_final_step %}
+  {{ progress_bar(4) }}
```

Comment:
similar comment for the number as before. It would be nice if we could use something like `QUESTION_SUBMITTED` or similar